dist/src/main/java/io/camunda/application/commons/backup/BackupConfig.java

```diff
@@ -65,10 +65,10 @@ && validRepoName(tasklistBackup.getRepositoryName())
   public ThreadPoolTaskExecutor getTaskExecutor() {
     final ThreadPoolTaskExecutor executor = new ThreadPoolTaskExecutor();
     executor.setCorePoolSize(1);
-    executor.setMaxPoolSize(1);
+    executor.setMaxPoolSize(8);
+    executor.setKeepAliveSeconds(60);
     executor.setThreadNamePrefix("webapps_backup_");
-    // TODO why just 6?
-    executor.setQueueCapacity(6);
+    executor.setQueueCapacity(4096);
```

Comment:
:question: Why do we explicitly set the queue anyway?

Reply:
by default it has no limit, so it's probably better to limit it

Reply:
Oh, I see Spring just puts it "unbounded" (sort of). Hm. I guess it all depends how we deal with rejected execution exceptions...

Reply:
Considering it was already setting it, then hopefully we do anyway :shrug: